models/migrations/migrations.go

```diff
@@ -369,9 +369,11 @@ var migrations = []Migration{
 	// v207 -> v208
 	NewMigration("Add webauthn table and migrate u2f data to webauthn", addWebAuthnCred),
```

Comment:
Does the migration compare against keys that were removed in the meantime? If not, wouldn't it be now a good idea to let 207 only create the table without writing actual data to speed up things a bit?

Reply:
OK I've no-oped that too.